src/api/types.ts

```diff
@@ -189,13 +189,45 @@ interface CompaniesHouseAddress {
   country: string | null;
 }
 
-interface CompanyHouseResponse extends SuccessResponse {
+interface CompaniesHouseAccountReferenceDate {
+  month: string;
+  day: string;
+}
+
+interface CompaniesHouseAccounts {
+  accounting_reference_date: CompaniesHouseAccountReferenceDate;
+}
+
+interface CompaniesHouseAPIAddress {
+  care_of: string | null;
+  premises: string | null;
+  address_line_1: string | null;
+  address_line_2: string | null;
+  locality: string | null;
+  region: string | null;
+  postal_code: string | null;
+  country: string | null;
```

Comment:
Could all of these be with question mark notation instead of `string | null`? e.g

```
premises?: string;
```